src/ServiceInformJobList.c

```diff
@@ -0,0 +1,168 @@
+//ServiceJobList.c
+
+// SPDX-FileCopyrightText: 2024, Yaskawa America, Inc.
+// SPDX-FileCopyrightText: 2024, Delft University of Technology
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "MotoROS.h"
+
+rcl_service_t g_serviceListInformJobs;
+
+ServiceListInformJobs_Messages g_messages_ListInformJobs;
+
+typedef motoros2_interfaces__srv__ListInformJobs_Response ListInformJobsResponse;
+
+
+static const size_t RESP_NAMES_LIST_INITIAL_SIZE = 1;
+static const size_t MAX_NUMBER_OF_JOBS_TO_REPORT = 1024;
+
+#define MSG_RC(x) (motoros2_interfaces__msg__InformJobCrudResultCodes__RC_ ## x)
```

Comment:
You're giving me flashbacks to MFC programming w/ these macros.

Reply:
:)